### DISCUSSION TITLE
export encryption

### DISCUSSION THREAD

**Discussion by sda17** [1Remote/1Remote#610] [Ideas]
**Is your feature request related to a problem? Please describe.**
Is it possible to set up the encryption of passwords or contents in the xxxx.prma file

**Describe the solution you'd like**
the encryption of passwords or contents in the xxxx.prma file or the encryption of the xxxx.prma file

Thanks you !!

**Comment by VShawn** [comment]
I means you can zip the exported file with a password.

**Comment by sda17** [comment]
someone could make an export and then look at the passwords contained in it, hence my request.

**Reply by VShawn** [reply]
sorry to hear that someone else has been able to access the data exporting on your personal computer. Generally, we believe that only trusted people should have access to your computer desktop.

IMO If you sharing your computer with untrusted one, then the `encrypted data export` cannot guarantee the security of the data.

Or If you only want to share the account/password to your teams, I believe this can help you: https://1remote.org/usage/team/team-sharing/

**Reply by majkinetor** [reply]
This can be fixed with this option:

![image](https://github.com/1Remote/1Remote/assets/85767/0e93cc5a-7c58-4eaf-bda5-0b5f5b1d0075)

With it, when "somebody else" tries to export, he is prompted to enter your OS password. To disable this option, you are again required to enter OS password.  This also protects viewing your password within connection configuration.

I think this solves your problem. You can protect your export after it is created with multiple methods, such as protected archive.

**Reply by VShawn** [reply]
yes credentials verification can also help

...I actually forgot about this feature

**Comment by sda17** [comment]
this solution suits me fine. Thank you for your answer and help.
